src/argparse.ts

```diff
@@ -2271,6 +2271,60 @@ export const CLI_ARGS = {
       '    ]\n' +
       '\n',
       group: 'Peer Services'
+    },
+    get_did_configuration: {
+      type: 'array',
+      items: [
+        {
+          name: 'blockstack_id',
+          type: 'string',
+          realtype: 'blockstack_id',
+          pattern: NAME_PATTERN + '|'+ SUBDOMAIN_PATTERN
+        },
+        {
+          name: 'domain',
+          type: 'string',
+          realtype: 'domain',
+          pattern: NAME_PATTERN + '|'+ SUBDOMAIN_PATTERN
+        },
+        {
+          name: 'owner_key',
+          type: 'string',
+          realtype: 'private_key',
+          pattern: `${PRIVATE_KEY_PATTERN}`
+        },
+      ],
+      minItems: 3,
+      maxItems: 3,
+      help: 'Creates a DID configuration for the given blockstack id and domain to create a link between both.' +
+      'The specification is define by the Decentralized Identity Foundation at https://identity.foundation/specs/did-configuration/\n' +
+      'The DID configuration should be placed in the json file ".well_known/did_configuration"' +
+      '\n'+
```

Comment:
Please insert another newline here -- `Example:` should have a line break between the document body and the runtime example.

Reply:
Done